src/test/java/com/uwetrottmann/tmdb2/TestData.java

```diff
@@ -1,44 +1,243 @@
-    String REVIEW_ISO_639_1 = "en";
-    Integer REVIEW_MEDIA_ID = 284052;
-    String REVIEW_MEDIA_TITLE = "Doctor Strange";
-    String REVIEW_MEDIA_TYPE = "Movie";
-
-    int MOVIE_WITH_COLLECTION_ID = 671;
-    String MOVIE_WITH_COLLECTION_TITLE = "Harry Potter and the Philosopher\'s Stone";
-
-    int MOVIE_COLLECTION_ID = 1241;
-    String MOVIE_COLLECTION_TITLE = "Harry Potter Collection";
-
-    String TVSHOW_TITLE = "Breaking Bad";
-    int TVSHOW_ID = 1396;
-    String TVSHOW_IMDB_ID = "tt0903747";
-    int TVSHOW_SEASON1_ID = 30272;
-    String EPISODE_IDMB_ID = "tt0959621";
-    int PERSON_ID = 1;
-    String PERSON_NAME = "George Lucas";
-    String PERSON_IMDB_ID = "nm0000184";
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+
+public class TestData {
+
+    private static final SimpleDateFormat JSON_STRING_DATE = new SimpleDateFormat("yyy-MM-dd");
+
+    public static Person testPerson = new Person();
+    public static BasePerson testBasePerson = (BasePerson) testPerson;
+    public static Date testPersonChangesStartDate;
+    public static Date testPersonChangesEndDate;
+
+    public static Person testPersonCast = new Person();
+    public static Person testPersonCrew = new Person();
+
+    public static Movie testMovie = new Movie();
+    public static BaseMovie testBaseMovie = (BaseMovie) testMovie;
+    public static Date testMovieChangesStartDate;
+    public static Date testMovieChangesEndDate;
+
+    public static Credit testCredit = new Credit();
+
+    public static Collection testCollection = new Collection();
+
+    public static Genre testMovieGenreAdventure = new Genre();
+    public static Genre testMovieGenreRomance = new Genre();
+    public static Genre testTvGenreDrama = new Genre();
+    public static Genre testTvGenreSciFi = new Genre();
+
+    public static Network testNetwork = new Network();
+
+    public static Review testReview = new Review();
+
+    public static Company testMovieCompany = new Company();
+    public static BaseCompany testProductionCompany = (BaseCompany) testMovieCompany;
+    public static Company testCompany = new Company();
+
+
+    public static TvShow testTvShow = new TvShow();
+    public static Date testTvShowChangesStartDate;
+    public static Date testTvShowChangesEndDate;
+
+    public static TvSeason testTvSeason = new TvSeason();
+    public static Date testTvSeasonChangesStartDate;
+    public static Date testTvSeasonChangesEndDate;
+
+    public static TvEpisode testTvEpisode = new TvEpisode();
+    public static Date testTvEpisodeChangesStartDate;
+    public static Date testTvEpisodeChangesEndDate;
+
+    public static BaseKeyword testKeyword = new BaseKeyword();
+
+    public static List testList = new List();
+
+    public static java.util.List<Integer> testListMovies = new java.util.ArrayList<>();
+
+    public static void initializeTestingIntegrityData() throws ParseException {
```

Comment:
This should all be constant data (static final). It is confusing that one has to call the initialize method before all the constants actually have a value, no? Though it works for now...